middlewares/authenticate.js

```diff
@@ -58,7 +58,7 @@ module.exports = async (req, res, next) => {
         })
 
         // add user data to `req.userData` for further use
-        req.userData = await users.fetchUser(decoded.userId)
+        req.userData = await users.fetchUser({ userId: decoded.userId })
```

Comment:
Destructure `decoded` it's being used at multiple places

Reply:
Yes sure

Reply:
Done with requested changes. Please review.

Reply:
Did you push your changes?
Can't see destructured `decoded` varialble